benchmark-service/benchmark/benchmark.py

```diff
@@ -0,0 +1,58 @@
+import os
+from pathlib import Path
+from typing import Any, AsyncIterator, Dict
+
+import aiohttp_jinja2
+import jinja2
+from aiohttp import web
+import logging
+
+app = web.Application()
+router = web.RouteTableDef()
+logging.basicConfig(level=logging.DEBUG)
+
+
+aiohttp_jinja2.setup(
+    app, loader=jinja2.ChoiceLoader([
+        jinja2.PackageLoader('benchmark', 'templates')
+    ]))
+
+
+@router.get("/healthcheck")
+async def healthcheck() -> web.Response:
+    return web.Response()
+
+
+@router.get('/{username}')
+async def greet_user(request: web.Request) -> web.Response:
+
+    context = {
+        'username': request.match_info.get("username", ""),
```

Comment:
We try to use single quotes where possible. There's a bunch of places in here that can be changed.